docs/DART_LAB/matlab/run_lorenz_96_inf.m

```diff
@@ -39,19 +39,20 @@
 % that variable, in any function, is visible to all the functions that declare
 % it as global.
 
-global TRUE_FORCING;
-global FORCING;
-global MODEL_SIZE;
-global DELTA_T;
+global TRUE_FORCING
+global FORCING
+global MODEL_SIZE
+global DELTA_T
+global MEAN_DIST
 
 LOG_FILE = strcat(mfilename, '.log');
 
 first_call_to_reset = true;
 
 atts = stylesheet; % get the default fonts and colors
 
-figWidth  = 990;   % in pixels
-figHeight = 820;    % in pixels
+figWidth  = 1000;    % in pixels
```

Comment:
Actually ... please make this 1024
on my machine, 1000 makes the text box above the 'Inf Sd.' gets clipped to simply 'Inf' ...

Reply:
Will do!